connectors/src/connectors/zendesk/temporal/workflows.ts

```diff
@@ -200,6 +207,30 @@ export async function zendeskSyncWorkflow({
   await saveZendeskConnectorSuccessSync({ connectorId });
 }
 
+/**
+ * Syncs the tickets updated since the last scheduled execution.
+ */
+export async function zendeskIncrementalSyncWorkflow({
```

Comment:
do you plan to add the `launchZendeskIncrementalSyncWorkflow` with the cron parameters in another pr?

Reply:
the plan in this PR is to execute `zendeskIncrementalSyncWorkflow` as a child workflow of the main `zendeskSyncWorkflow`

Reply:
Did not know about the workflow starting right after the previous one, thanks for the insight!
We might want to run the incremental sync in another workflow than the one for the initial sync to make sure it doesn't have to wait it out then.
I will keep that in mind and monitor in that direction to see if we opt for splitting in 2 workflows 👍